src/blocks/BlockFactory.tsx

```diff
@@ -100,6 +101,7 @@ export default class BlockFactory {
       case "Pickrr": return <PickrrFeedComponent model={model} update={update} />;
       case "Riddlequest": return <RiddlequestFeedComponent model={model} update={update} />;
       case "Color": return <ColorFeedComponent model={model} update={update} />;
+      case "Seam": return <SeamFeedComponent model={model} update={update} />;
```

Comment:
Let's rename this miniapp, because just `Seam` is too generic -- "MemeGen" is good